mod/metric_answer_lookup/set/type_plus_right/metric_value/value.rb

```diff
@@ -1,3 +1,5 @@
+card_accessor :unknown
```

Comment:
toggle cards have a very limited api. I have to override the editor view to get a label